lib/paubox/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Paubox
-  VERSION = '0.3.2'
```

Comment:
This is really bizarre, and why is this seemingly crucial hotfix dead for 10 months now?  Is Paubox dead?